### PR TITLE
[NUI][ATSPI] Accessibility added to Loading component

### PR DESCRIPTION
**Pull request by a-bartak**
… components.

### API Changes ###

 - ACR: None

### CODE REVIEW

**Comment by huiyueun** [comment]
Please add [NUI] tag to title.

File: src/Tizen.NUI.Components/Controls/Loading.cs [hunk 1]
```diff
@@ -114,6 +116,7 @@ public Loading(string style) : base(style)
         /// <since_tizen> 8 </since_tizen>
         public Loading(LoadingStyle loadingStyle) : base(loadingStyle)
         {
+            Initialize();
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/Loading.cs:119]: please check the warning below. => line 135 "Properties should not return arrays"
returning array is not good in public method, if there is not other way, you may use read only collection as return type.
reference : https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1819

Comment (a-bartak) [reply src/Tizen.NUI.Components/Controls/Loading.cs:119]: The warning mentioned concerns code that wasn't changed in this pull request.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Comment by dongsug-song** [comment]
@a-bartak 
Hello, 
could you resolve "This branch is out-of-date with the base branch" error?
thank you.

**Review by lol-github — CHANGES_REQUESTED** [review]
all usages of legacy API (AccessibilityManager) should be removed or replaced with the new API

**Comment by dongsug-song** [comment]
@a-bartak 
Hello, 
could you resolve the error "This branch is out-of-date with the base branch" below? 
(by this error, I can not "Rebase and merge")
or could you upload new PR again?

**Review by lol-github — APPROVED** [review]
LGTM

**Comment by dongsug-song** [comment]
@a-bartak 
Hello, 
now I put the \*\***NO MERGE**\*\* tag on thie PR.
you can remove \*\***NO MERGE**\*\* tag for yourself when this PR is fixed and ready to be merged.
(\*\***NO MERGE**\*\* tag has no restriction to submit PR or to editing this patch but just for notification and marking.
thank you.

**Review by Seoyeon2Kim — CHANGES_REQUESTED** [review]
Dear @a-bartak and @lol-github ,
Loading class is a component which indicates that the current state is waiting for the next.
(https://github.com/dalihub/nui-demo/blob/master/wearable-samples/Indicator/ActivityIndicator(Small)/res/activityIndicator.gif)
I'm just wondering if it's correct that the role is ProgressBar or not.
As you know, we have `Progress` class separately.

If 'ProgressBar' of Accessibility role matches `Loading` class, could you let me know to refer to the entire list to see what it is?

**Comment by lol-github** [comment]
> Dear @a-bartak and @lol-github ,
> Loading class is a component which indicates that the current state is waiting for the next.
> (https://github.com/dalihub/nui-demo/blob/master/wearable-samples/Indicator/ActivityIndicator(Small)/res/activityIndicator.gif)
> I'm just wondering if it's correct that the role is ProgressBar or not.
> As you know, we have `Progress` class separately.
> 
> If 'ProgressBar' of Accessibility role matches `Loading` class, could you let me know to refer to the entire list to see what it is?

Dear @Seoyeon2Kim,
we decided to finally assign the role of ProgressBar as it converge with screen reader's UX Guideline (available here: https://xd.adobe.com/view/1bc8f657-f2ab-4e3f-6bf5-d5b6efdbc8cc-6df9/   password: 6TizenGUI). Even though there is no UI element literally called "Loading" there is a UI element called 2.4.2.1  ActivityIndicator (on slide 118) which IMHO perfectly match the Loading widget. UX guideline states that in such case the "Inprogress" reading should be heard, which is identical to the case of undetermined 2.4.1.3 ProgressBar (slide 116)

**Comment by Seoyeon2Kim** [comment]
> > Dear @a-bartak and @lol-github ,
> > Loading class is a component which indicates that the current state is waiting for the next.
> > (https://github.com/dalihub/nui-demo/blob/master/wearable-samples/Indicator/ActivityIndicator(Small)/res/activityIndicator.gif)
> > I'm just wondering if it's correct that the role is ProgressBar or not.
> > As you know, we have `Progress` class separately.
> > If 'ProgressBar' of Accessibility role matches `Loading` class, could you let me know to refer to the entire list to see what it is?
> 
> Dear @Seoyeon2Kim,
> we decided to finally assign the role of ProgressBar as it converge with screen reader's UX Guideline (available here: https://xd.adobe.com/view/1bc8f657-f2ab-4e3f-6bf5-d5b6efdbc8cc-6df9/ password: 6TizenGUI). Even though there is no UI element literally called "Loading" there is a UI element called 2.4.2.1 ActivityIndicator (on slide 118) which IMHO perfectly match the Loading widget. UX guideline states that in such case the "Inprogress" reading should be heard, which is identical to the case of undetermined 2.4.1.3 ProgressBar (slide 116). Moreover, there is no ActivityIndicator role in ATSPI standard so we have to use one of the available roles, like ProgressBar.

Dear @lol-github ,
Okay, now I understand it.
Thank you for your explanation.
ATSPI should read "In Progress" like ProgressIndicator's estimating(indeterminate) mode.
Also, there's no ActivityIndicator role in ATSPI standard. Let's go to this PR now. :)

**Review by Seoyeon2Kim — APPROVED** [review]
Approved.
After updating this PR because of 'This branch is out-of-date with the base branch' and removing 'No Merge' tag, you can merge it by yourself. :)

**Comment by Seoyeon2Kim** [comment]
Dear @a-bartak , can we remove 'NO MERGE' tag for this PR?

**Comment by a-bartak** [comment]
> Dear @a-bartak , can we remove 'NO MERGE' tag for this PR?

Yes, of course.

**Pull request merged**